src/events/pollVotationResults.js

```diff
@@ -10,11 +10,15 @@ module.exports = {
       return;
     }
 
+    if (!embeds[0]?.fields) {
+      return;
+    }
+
     const userMentionField = embeds[0]?.fields.find(
       (field) => field.name === 'poll_question_text'
     );
     const parts = userMentionField?.value.split('|').map((part) => part.trim());
-    const userMentioned = `<@${parts[1]}>`;
+    const userMentioned = parts?.[1] ? `<@${parts[1]}>` : null;
```

Comment:
With the change on line 13 I think this part is not needed
```suggestion
    const userMentioned = `<@${parts[1]}>` ;
```

If you have doubts maybe you could use another early return here, so execution will stop instead of continue but create invalid user id like `<@null>`.

```suggestion
    if (!Array.isArray(part)) return;
    
    const userMentioned = `<@${parts[1]}>`;
```

Reply:
Completed